### PR TITLE
Validation for creation of new road signs

### PR DESCRIPTION
**Pull request by nbittich**
Can you please have a look to the changes? I made the validations only for the new road signs, if the pull request is accepted I will do it for the others as well in a separate branch. Thanks!

### CODE REVIEW

File: app/templates/road-sign-concepts/new.hbs [hunk 1]
```diff
@@ -7,51 +7,75 @@
...
 
   <form
-    class="au-c-form au-o-layout au-u-padding-top-large"
+          class="au-c-form au-o-layout au-u-padding-top-large"
     {{on "submit" this.createRoadSignConcept}}
   >
     <p>
-      <AuLabel for="image-url">{{t "newRoadSign.imageUrl"}}</AuLabel>
-      <AuInput type="url" id="image-url" required="required" {{on "input" (fn this.setRoadSignConceptValue "image")}} />
+      <AuLabel for="image-url">{{t "newRoadSign.imageUrl"}}&nbsp;<AuPill>{{t "newRoadSign.required"}}</AuPill>
```
Comment (Windvis) [inline app/templates/road-sign-concepts/new.hbs:17]: Maybe you can wrap everything inside a let block:

```hbs
{{#let (is-empty @model.newRoadSignConcept.image) as |isInvalid|}}
{{!-- use isInvalid --}}
{{/let}}
```

That would reduce the duplication and make it more clear what's happening?

Comment (Windvis) [reply app/templates/road-sign-concepts/new.hbs:17]: I think the AuLabel also has an `@error` argument, but maybe that's overkill?

File: app/templates/road-sign-concepts/new.hbs [hunk 2]
```diff
@@ -7,51 +7,75 @@
     </LinkTo>
   </BreadcrumbsItem>
 
-  <NavigationBar />
+  <NavigationBar/>
```
Comment (Windvis) [inline app/templates/road-sign-concepts/new.hbs:10]: What formatter are you using? It seems to do some weird thing imo :smile:

File: translations/en-us.yaml [hunk 3]
```diff
@@ -45,6 +45,7 @@ newMeasureConcept:
   roadSignConcept: Road Sign
   noData: No results
   create: Create new measure
+  required: Required
```
Comment (Windvis) [inline translations/en-us.yaml:48]: I think these don't need to be duplicated. I can't imagine a reason why you want multiple variantions for this. I know you added them to be consistent though :smile:.

**Review by Windvis — CHANGES_REQUESTED** [review]

File: app/templates/road-sign-concepts/new.hbs (hunk 2 above)
Comment (nbittich) [reply app/templates/road-sign-concepts/new.hbs:10]: yeah I was thinking about the same, probably my IDE is messing up. I will commit from the command line next so it should fix it

File: translations/en-us.yaml (hunk 3 above)
Comment (nbittich) [reply translations/en-us.yaml:48]: well i proposed to have something like 
common:
   required: xxx
   
  But the team kinda ignore me so I'm following the trends :p

File: app/templates/road-sign-concepts/new.hbs (hunk 1 above)
Comment (nbittich) [reply app/templates/road-sign-concepts/new.hbs:17]: there is no @error on the aulabel level. I tried it and it doesn't do anything 
image

File: translations/en-us.yaml (hunk 3 above)
Comment (Windvis) [reply translations/en-us.yaml:48]: Yea, I think Anita created a utilities top level thing in her PR. Maybe you can check that out.

```yml
utilities:
  required: Required
```

**Pull request merged**